assets/javascripts/discourse/initializers/hook-composer.js.es6

```diff
@@ -47,17 +47,17 @@ export default {
 
       @on("init")
       initEncrypt() {
-        this.updateEncryptProperties();
+        this.updateEncryptProperties(true);
```

Comment:
true is a bit surprising here... `this.updateEncryptProperties({newMessage: true})` though more verbose is far more easy to follow.